### DISCUSSION TITLE
How to set the width of the connection

### DISCUSSION THREAD

**Discussion by Feliks9527** [bcakmakoglu/vue-flow#1631] [Q&A]
![image](https://github.com/user-attachments/assets/70fd9dca-15f6-45ab-9f1a-ec7946b27709)

**Comment by bcakmakoglu** [comment]
Would you be so kind as to take the time to actually formulate a whole question instead of just posting a picture?
It would be very much appreciated and helps me in knowing *what exactly you are trying to do*.
